aiida_registry/test_install.py

```diff
@@ -179,6 +181,16 @@ def filter_entry_points(process_metadata, entrypoints):
     return filtered_metadata
 
 
+def add_warnings(metadata):
+    """Add fetch warnings to the data object."""
+    plugins_warnings = REPORTER.plugins_warnings
+
+    for name, warnings in plugins_warnings.items():
+        metadata["plugins"][name]["warnings"] = warnings
```

Comment:
Hi @AhmedBasem20, the page build is successful but there's a problem here that the warnings are generated in two steps and test install warning will override the previous warnings. You can append the new message is it was there already.

Reply:
BTW, I can run `aiida-registry test-install` on my Apple M1 Macbook, use the new docker image which has arm64 supported.

```bash
# install colima
brew install colima
# Use new docker sock, the python docker API is use DOCKER_HOST to find the docker.sock
echo 'export DOCKER_HOST="unix://$HOME/.colima/default/docker.sock"' >> ~/.zprofile
# cd into the folder and run 
aiida-registry test-install --container-image aiidateam/aiida-core-with-services:edge
```

Reply:
> BTW, I can run aiida-registry test-install on my Apple M1 Macbook, and use the new docker image which has arm64 supported.


This is exactly what I need for this task😅 Thanks for the reminder!

Reply:
Yes, I know, I also try to reproduce it and find it takes so long to see what happened. It is nearly impossible to debug over action output 😄

Reply:
Also I see that those `test-install` messages are actually errors not warnings, should we keep them with warnings or separate them? (i.e. plugin["warnings"], plugin["errors"])

Reply:
The "permission denied" error we saw yesterday was because the UID of my workstation is `1001` while the container has `1000` as default, so the file can not be written to the folder. I added the write permission and it is all good.

Reply:
> Yes, I know, I also try to reproduce it and find it takes so long to see what happened. It is nearly impossible to debug over action output 😄

I have a very naive way on debugging over github actions by breaking the `test-install` loop after five plugins for example (https://ahmedbasem20.github.io/aiida-registry/aiida-alloy). This is temporarily faster but I'll defiantly try to set the environment properly.